src/components/DoctorsPage/DoctorsCard/Schedule/Schedule.tsx

```diff
@@ -21,9 +26,17 @@ const initialState = {
 const reducer = (state: IScheduleInitialState, action: ActionType): IScheduleInitialState => {
   switch (action.type) {
     case ScheduleActionTypes.SET_CHOICE_WORK_TIME:
-      return { ...state, choiceWorkTime: { date: action.payload.date, time: action.payload.time } };
+      return {
+        ...state,
+        viewSuccessMessage: false,
+        choiceWorkTime: {
+          date: action.payload.date,
+          time: action.payload.time,
+          _id: action.payload._id,
```

Comment:
keys are the same, what for listing them?

Reply:
changed to spread